contrib/babelfishpg_common/sql/geography.sql

```diff
@@ -430,6 +430,62 @@ CREATE OR REPLACE FUNCTION sys.STIsValid(sys.GEOGRAPHY)
         AS '$libdir/postgis-3','isvalid'
         LANGUAGE 'c' IMMUTABLE STRICT PARALLEL SAFE;
 
+-- HasZ
```

Comment:
Kindly update the PR title to a bit more descriptive. For eg: 
TDS receiver and Z/M functions support for spatial datatypes